www/templates/default/html/manager/Calendar.tpl.php

```diff
@@ -1,31 +1,149 @@
 <?php
-    $events = $context->getCategorizedEvents();
+    $events = $context->getEvents();
+    $categorized_events = $context->getCategorizedEvents();
+    $total_pages = NULL;
+    switch ($context->tab) {
+        case 'pending':
+            $total_pages = ceil(count($categorized_events['pending']) / 10);
+            break;
+        case 'upcoming':
+            $total_pages = ceil(count($categorized_events['posted']) / 10);
+            break;
+        case 'past':
+            $total_pages = ceil(count($categorized_events['archived']) / 10);
+            break;
+        default:
+            $total_pages = 1;
+    }
 ?>
 <ul class="wdn_tabs">
-    <li><a href="#pending">Pending (<?php echo count($events['pending']); ?>)</a></li>
-    <li><a href="#upcoming">Upcoming (<?php echo count($events['posted']); ?>)</a></li>
-    <li><a href="#past">Past (<?php echo count($events['archived']); ?>)</a></li>
+    <li class="<?php if ($context->tab == 'pending') echo 'selected'; ?>"><a href="<?php echo ($context->tab == 'pending' ? '#pending' : '?tab=pending'); ?>">Pending (<?php echo count($categorized_events['pending']); ?>)</a></li>
```

Comment:
Its not super necessary, but it would be nice to have a vanity URL for these like `/pending` or `/upcoming` instead of `tab=pending`.